l10n_it_vat_registries/wizard/print_registro_iva.py

```diff
@@ -37,6 +37,16 @@ class WizardRegistroIva(models.TransientModel):
         string="Year for Footer", help="Value printed near number of page in the footer"
     )
 
+    report_order_by = fields.Selection([('date_name','Date Name'),
+                                        ('type_date','Invoice Type Date'),
+                                        ('type_date_name','Invoice Type Date Name')],'Print Layout')
```

Comment:
Puoi mettere `date_name' come default, in modo da mantenere il comportamento precedente come predefinito?

Reply:
funziona già cosi se guardi il codice, se uno non mette niente prende date_name

```
    def getMatchOrderBy(self):
        if self.report_order_by == 'type_date':
            return "move_type, date"
        if self.report_order_by == 'type_date_name':
            return "move_type, date, name"
        return "date, name"
```


fammi sapere

ciao

Matteo

Reply:
Ok.
Puoi aggiungere le virgole nei valori della selezione in modo da separare i campi?
Invece di `Print Layout` userei tipo `Print order`.
Grazie